src/java.base/share/classes/java/lang/classfile/AnnotationElement.java

```diff
@@ -22,20 +22,24 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
-package jdk.internal.classfile;
+package java.lang.classfile;
 
 import java.lang.constant.ClassDesc;
 
-import jdk.internal.classfile.constantpool.Utf8Entry;
+import java.lang.classfile.constantpool.Utf8Entry;
 import jdk.internal.classfile.impl.AnnotationImpl;
 import jdk.internal.classfile.impl.TemporaryConstantPool;
+import jdk.internal.javac.PreviewFeature;
 
 /**
```

Comment:
shouldn't we have the preview header applied to all these compilation units?:

```
{@preview Associated with pattern matching for instanceof, a preview feature of the Java language.
```

Reply:
I'm not aware of preview javadoc tag, could you point me to an example?

Reply:
nvm, I was rechecking JEP 12 this javadoc should now be generated automatically so we are good